src/Symfony/Component/Validator/ConstraintViolation.php

```diff
@@ -18,27 +18,27 @@
  */
 class ConstraintViolation implements ConstraintViolationInterface
 {
-    private $message;
-    private $messageTemplate;
-    private $parameters;
-    private $plural;
-    private $root;
-    private $propertyPath;
-    private $invalidValue;
-    private $constraint;
-    private $code;
-    private $cause;
+    private string|\Stringable $message;
+    private ?string $messageTemplate;
+    private array $parameters;
+    private ?int $plural;
+    private mixed $root;
+    private ?string $propertyPath;
+    private mixed $invalidValue;
+    private ?Constraint $constraint;
+    private mixed $code;
```

Comment:
the code constructor argument is typed as ?string

Reply:
Good catch!